tests/goldenfiles/divided_highway_split.txt

```diff
@@ -1,10 +1,10 @@
 TurnID(Lane #2, Lane #6, Intersection #0) is a SharedSidewalkCorner
-TurnID(Lane #4, Lane #9, Intersection #0) is a Right
+TurnID(Lane #3, Lane #9, Intersection #0) is a Right
```

Comment:
For future reference, you can do something like `./import.sh --oneshot=tests/input/left_turn_and_bike_lane.osm` and then view the map normally to see what's going on with these unit tests. For all 3 of these, I manually verified the turns before and after this PR, and this PR improves all of them (by human judgment). The goldenfile will prevent regressions.